src/modules/esl-utils/dom/rect.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * A Rect describes the size and position of a rectangle.
+ * */
+export class Rect {
+  /**
+   * The X coordinate of the Rect's origin (top-left corner of the rectangle).
+   * */
+  public x = 0;
+  /**
+   * The Y coordinate of the Rect's origin (top-left corner of the rectangle).
+   * */
+  public y = 0;
+  /**
+   * The width of the Rect.
+   * */
+  public width = 0;
+  /**
+   * The height of the Rect.
+   * */
```

Comment:
Cosmetic: it's preferable to use compact one-line comments. Basically no problems with the just an agreement in bounds of the project
`/** One line comment */`

Reply:
done